lm75bd/lm75bd.c

```diff
@@ -27,8 +30,19 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 }
 
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
-  /* Implement this driver function */
-  
+  uint8_t tmp_reg_pointer = LM75BD_TEMP_REG;
+  i2cSendTo(devAddr, &tmp_reg_pointer, 1);
+  uint8_t buffer[] = {0x00, 0x00};
+  i2cReceiveFrom(devAddr, &buffer[0], 2);
```

Comment:
This is equivalent to `i2cReceiveFrom(devAddr, buffer, 2)`. We tend to do it this way since it's a bit cleaner.